oojs.js

```diff
@@ -10,3 +10,23 @@
   // Have an artist property that is determined by some input.
 
 // Create an instance of the Album class.
+
+class Playlist{
+  constructor(title, song){
+    this.title = title;
+    this.song = song;
```

Comment:
`this.song` should be plural and initialized as an array, and if you're passing an optional `song` into your constructor, it needs to be pushed onto that array if available
```js
this.songs = [];
if(song) this.songs.push(song);
```